mlflow/R/mlflow/.Rprofile

```diff
@@ -1,3 +1,7 @@
 # https://packagemanager.rstudio.com provides pre-compiled package binaries for Linux
 # that can be installed significantly faster than uncompiled package sources.
-options(repos = c(REPO_NAME = "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"))
+if (Sys.which("lsb_release") != "") {
+    ubuntu_codename <- tolower(system("lsb_release -cs", intern = TRUE))
```

Comment:
Is it worth putting in the option `show.output.on.console` to FALSE for Windows or is that pointless?

Reply:
Offline discussion: we don't need it for now since we run this on Ubuntu.